contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBasePushFilterIntoScan.java

```diff
@@ -125,7 +125,7 @@ protected void doPushFilterToScan(final RelOptRuleCall call, final FilterPrel fi
     final ScanPrel newScanPrel = ScanPrel.create(scan, filter.getTraitSet(), newGroupsScan, scan.getRowType());
 
     // Depending on whether is a project in the middle, assign either scan or copy of project to childRel.
-    final RelNode childRel = project == null ? newScanPrel : project.copy(project.getTraitSet(), ImmutableList.of((RelNode)newScanPrel));;
+    final RelNode childRel = project == null ? newScanPrel : project.copy(project.getTraitSet(), ImmutableList.of((RelNode)newScanPrel));
```

Comment:
Could you please add space: `(RelNode)newScanPrel` -> `(RelNode) newScanPrel`

Reply:
Removed the cast. IntelliJ said it was unnecessary.